compiler/rustc_session/src/session.rs

```diff
@@ -631,9 +563,6 @@ impl Session {
     pub fn verify_llvm_ir(&self) -> bool {
         self.opts.debugging_opts.verify_llvm_ir || option_env!("RUSTC_VERIFY_LLVM_IR").is_some()
     }
-    pub fn borrowck_stats(&self) -> bool {
-        self.opts.debugging_opts.borrowck_stats
```

Comment:
Can you remove the borrowck_stats option too?

Reply:
Done, good catch!